### PR TITLE
Two FAQ updates

### PR DESCRIPTION
**Pull request by grinapo**
- about audit trail (without any tech details though)
- about how changes are seen, based on the forum discussion

Feel free to correct or edit! Thanks!

### CODE REVIEW

**Comment by imsodin** [comment]
I directly committed some language changes, github review isn't very practical to propose small changes to text.

Maybe neighbour should be replaced with remote device to be consistent with the GUI?

Are you sure about the second FAQ entry? I am a a bit confused by the part about multiple remote devices. They should already be in a consistent state so we normally don't need to decide between remote devices, just between the received and local index.

**Comment by grinapo** [comment]
@imsodin FAQ: yes, but if anyone wants to use it any other way: feel free. 

About consistency: they are not necessarily in sync; for example when there are two groups of nodes which were split, doing their changes, and I connect them both they will both believe their latest is The Reality™ while I will see that either one of them is newer or they're conflicting. I will be the one who actually pull both groups in sync with the rest of the mesh (if I undesrstand correctly).
This entry was born from the questions about reliability of the "originating node" in audit log and the realisation that the protocol "collapses" changes between last and current state (so "non-last changes"  will not be visible in local audit log either). Phrasing that is not easy, and I'm an outsider as well, so every input is most welcome.

File: users/faq.rst [hunk 1]
```diff
@@ -405,6 +405,32 @@ creates ``syncthing.log`` in Syncthing's home directory (run ``syncthing
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The audit log (and the ``Global Changes`` window) see the changes that your
+Syncthing sees. When Syncthing is continuously connected it usually sees every change
+happening immediately and thus knows which node initiated the change. 
+When topology gets complex or when your node reconnects after some time offline, 
+Syncthing synchronises with its neighbours: It gets the latest synchronised state 
+from the neighbour, which is the *result* of all the changes between the last 
+known state (before disconnect or network delay) and the current state at the 
+neighbour, and if there were updates, deletes, creates, conflicts, which were
```
Comment (ProactiveServices) [inline users/faq.rst]: IMO this could be succinctly described by something more like "...which is the result of all changes since last connection, not the individual changes which led to the current state." I feel it's a little too wordy as it is.

Comment (grinapo) [inline users/faq.rst]: @ProactiveServices fine with me, I'm not a native speaker.

**Comment by calmh** [comment]
@st-review merge it

users/faq: Describe global changes and auditing

**Comment by st-review** [comment]
:ok_hand: Merged as ed81a8726322d729b25251385a8b0a9fe20c5389. Thanks, @grinapo!
